Sources/Defaults/Defaults+iCloud.swift

```diff
@@ -193,6 +193,7 @@ extension Defaults.iCloud {
 }
 
 private enum SyncStatus {
+	case abort
```

Comment:
```suggestion
	case aborted
```

?

And should be the last case.